thermalnetwork/network.py

```diff
@@ -175,52 +175,90 @@ def convert_features(self, json_data):
                 logger.debug(f"matching_ghe: {matching_ghe}\n")
-                if "ghe_geometric_params" not in matching_ghe:
-                    # convert detailed geometry coordinates to meters
-                    lat_long_polys = feature["geometry"]["coordinates"]
-                    origin_lon_lat = lower_left_point(lat_long_polys[0])
-                    convert_facs = meters_to_long_lat_factors(origin_lon_lat)
-                    ghe_polygons = []
-                    for poly in lat_long_polys:
-                        coords = lon_lat_to_polygon(poly, origin_lon_lat, convert_facs)
-                        ghe_polygons.append(coords)
-                    ghe_polygons = rotate_polygon_to_axes(ghe_polygons)
-                    # set geometric constraints to be dictated by the polygons
-                    geometric_constraints["polygons"] = ghe_polygons
-                    min_pt = lower_left_point(ghe_polygons[0])
-                    max_pt = upper_right_point(ghe_polygons[0])
-                    geometric_constraints["length"] = max_pt[0] - min_pt[0]
-                    geometric_constraints["width"] = max_pt[1] - min_pt[1]
-                else:  # use the old ghe_geometric_params
-                    length = matching_ghe["ghe_geometric_params"]["length_of_ghe"]
-                    width = matching_ghe["ghe_geometric_params"]["width_of_ghe"]
-                    geometric_constraints["length"] = length
-                    geometric_constraints["width"] = width
+                # if "ghe_geometric_params" not in matching_ghe:
+                #     # convert detailed geometry coordinates to meters
+                #     lat_long_polys = feature["geometry"]["coordinates"]
+                #     origin_lon_lat = lower_left_point(lat_long_polys[0])
+                #     convert_factors = meters_to_long_lat_factors(origin_lon_lat)
+                #     ghe_polygons = []
+                #     for poly in lat_long_polys:
+                #         coords = lon_lat_to_polygon(poly, origin_lon_lat, convert_factors)
+                #         ghe_polygons.append(coords)
+                #     ghe_polygons = rotate_polygon_to_axes(ghe_polygons)
+                #     # set geometric constraints to be dictated by the polygons
+                #     geometric_constraints["polygons"] = ghe_polygons
+                #     min_pt = lower_left_point(ghe_polygons[0])
+                #     max_pt = upper_right_point(ghe_polygons[0])
+                #     geometric_constraints["length"] = max_pt[0] - min_pt[0]
+                #     geometric_constraints["width"] = max_pt[1] - min_pt[1]
+                # else:  # use the old ghe_geometric_params
+                #     length = matching_ghe["ghe_geometric_params"]["length_of_ghe"]
+                #     width = matching_ghe["ghe_geometric_params"]["width_of_ghe"]
+                #     geometric_constraints["length"] = length
+                #     geometric_constraints["width"] = width
```

Comment:
@chriswmackey here's another place where I commented out lat/lon to coodinates, for now.